.github/CONTRIBUTING.md

```diff
@@ -114,14 +109,13 @@ fits with the scope and aims of the project. It's up to _you_ to make a strong
 case to convince the project's developers of the merits of this feature. Please
 provide as much detail and context as possible.
 
-
 ## Pull requests
 
 Good pull requests—patches, improvements, new features—are a fantastic
 help. They should remain focused in scope and avoid containing unrelated
 commits.
 
-**Please ask first** before embarking on any **significant** pull request (e.g.
+**Please [ask](https://github.com/twbs/bootstrap/issues/new?assignees=&labels=feature&template=feature_request.yml&title=Suggest+a+new+feature) first** before embarking on any **significant** pull request (e.g.
```

Comment:
This is not always true since a **significant** pull request could come from a new feature, a refactoring, an issue, a discussion, etc.